src/modules/navigation/NavContent.jsx

```diff
@@ -6,9 +6,10 @@ import Circle from 'cozy-ui/transpiled/react/Circle'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import OpenWithIcon from 'cozy-ui/transpiled/react/Icons/Openwith'
 import { NavIcon, NavText } from 'cozy-ui/transpiled/react/Nav'
+import Typography from 'cozy-ui/transpiled/react/Typography'
 import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
 
-const NavContent = ({ icon, badgeContent, label, external }) => {
+const NavContent = ({ icon, badgeContent, label, sharedDrives }) => {
```

Comment:
Here I will also still use external as this determines whether the external icon appears. But I prefer this if to separate cases 👍

Reply:
agreed